docs/_source/guides/features/semantic-search.ipynb

```diff
@@ -0,0 +1,1292 @@
+    "\n",
+    "Vector search leverages machine learning to capture rich semantic features by embedding items (text, video, images, etc.) into a vector space, which can be then used to find \"semantically\" similar items.\n",
+    "\n",
+    "In this guide, you'll find how to:\n",
+    "\n",
+    "- Setup your Elasticsearch or Opensearch endpoint with vector search support.\n",
+    "- Encode text into vectors for Argilla records.\n",
+    "- Use semantic search.\n",
+    "\n",
+    "The next section gives a general overview about how semantic search works in Argilla."
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "b648c176-0442-4905-b6e7-ed1056fbf83e",
+   "metadata": {},
+   "source": [
+    "## How it works\n",
+    "Semantic search in Argilla works as follows:\n",
+    "\n",
+    "1. One or several vectors can be included in the `vectors` field of Argilla Records. The `vectors` field accepts a dictionary where `keys` represent names and `values` the actual vectors. This is the case because certain use cases might require using several vectors. \n",
+    "2. The vectors are stored at indexing time, once the records are logged with `rg.log`. \n",
+    "3. If you have stored vectors in your dataset, you can use the semantic search feature in Argilla UI or the `vector` param in the `rg.load` method of the Python Client.\n",
+    "\n",
+    "In future versions, embedding services might be developed to facilitate steps 1 and 2 and associate vectors to records automatically. \n",
+    "\n",
+    "\n",
+    "<div class=\"alert alert-info\">\n",
+    "\n",
+    "Note\n",
+    "    \n",
+    "It's completely up to the user which encoding or embedding mechanism to use for producing these vectors. In the \"Encode text fields\" section of this document you will find several examples and details about this process, using open source libraries (e.g., Hugging Face) as well as paid services (e.g., Cohere or OpenAI).\n",
+    "\n",
+    "Currently, Argilla uses vector search only for searching similar records (nearest neighbours) of a given vector. This can be leveraged from Argilla UI as well as the Python Client. In the future, vector search could be leveraged as well for free text queries using Argilla UI.\n",
+    "    \n",
+    "</div>\n",
+    "\n",
+    "\n",
+    "## Setup Elasticsearch or Opensearch with vector search support\n",
+    "\n",
+    "In order to use this feature you should use Elasticsearch at least version `8.5.x`or Opensearch `2.3.0`. We provide pre-configured docker-compose files in the root of Argilla's [Github repository](https://github.com/argilla-io/argilla).\n",
+    "\n",
+    "<div class=\"alert alert-warning\">\n",
+    "\n",
+    "Warning\n",
+    "\n",
+    "If you had Argilla running with Elasticsearch 7.1.0 you need to migrate to at least version 8.5.x. Please check the section \"[Migrating from Elasticsearch 7.1.0 to 8.5.x](#migrating-from-elasticsearch-710-to-85x)\"\n",
```

Comment:
The section link does not work

Reply:
Don't remember adding this link ref. Could you take a look @davidberenstein1957 ,  I'll not tackle this this week

Reply:
I resolved this by adding casing.